src/isomorphic/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -591,11 +591,13 @@ describe('ReactPropTypes', function() {
 
   describe('OneOf Types', function() {
     it('should fail for invalid argument', function() {
-      typeCheckFail(
-        PropTypes.oneOf('red', 'blue'),
-        'red',
-        'Invalid argument supplied to oneOf, expected an instance of array.'
-      );
+      spyOn(console, ['error']);
```

Comment:
Can you change this to `spyOn(console, 'error')` like the rest of the codebase does?